sh/run.sh

```diff
@@ -14,7 +14,7 @@ if [ -z "$(ls -A /etc/my.cnf.d/* 2> /dev/null)" ]; then
         -e '/^innodb/d' /etc/my.cnf.d/my.cnf
 fi
 
-MYSQLD_OPTS="--user=mysql"
+MYSQLD_OPTS="--user=mysql --port=${MYSQL_PORT} ${MYSQL_EXT_OPTS}"
```

Comment:
My idea here was to override the port via a config instead of having to modify the shell script. Does that make sense?

Reply:
Some cases would create mutiple mariadb instances within the same box. A configurable port option would be necessary.
And with your design, leave port config in environment-variable is more conveniant. since the mariadb is *simplified* .